steps/telegram-notify/1.0.2/assets/icon.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```

Comment:
Icon should be on the path: `steps/telegram-notify/assets/icon.svg`; without version in the path.

Reply:
@lpusok im not sure where that came from, on my local repo i dont see any version path :/
The icon is on the root of the dir under assets :/
How can i change that?

Reply:
It is in your steplib fork (created by bitrise :share command): https://github.com/mcaldas/bitrise-steplib/tree/telegram-notify-1.0.4/steps/telegram-notify

Reply:
I recommend to try to start the share process again.

Reply:
@lpusok tried again and still got that icon stuck there :/
Any other suggestion?

Reply:
Can you try to remove 1.0.2 directory from the forked steplib repo manually? Or remove the complete fork repo to be sure.